src/library/classes.lisp

```diff
@@ -31,6 +31,19 @@
         ((Tuple (GT) (GT)) True)
         (_                 False))))
 
+  (define-instance (Ord Ord)
```

Comment:
Seems a little odd if you ask me. Does Haskell or Rust do this?

Reply:
[rust does](https://doc.rust-lang.org/stable/std/cmp/enum.Ordering.html#impl-Ord)

Reply:
[haskell does too](https://hackage.haskell.org/package/base-4.16.0.0/docs/Prelude.html#t:Ordering)